### ISSUE TITLE
small issue in documentation

### ISSUE THREAD

**Issue by koenhandekyn** [paperjs/paper.js#768]
on the features page at the bottom, the point and size assignments seem to be mixed up.

**Comment by lehni** [comment]
No, they're not : ) Read the text above, it says:

> Note that points are converted to sizes on the fly when required, and vice versa.

But I guess that's too cofusing?

**Comment by koenhandekyn** [comment]
oops :) i get it.  but it’s indeed a bit confusing. i thought you where pointing out the conversion from json alone.

related to conversion / operations, i found the fact that you covert from arrays quite handy and this might be more relevant.. things like "var other = (point1+[+10,+10])*[0.5,1]”. as far as i’ve seen this is not yet illustrated in the docs

**Comment by lehni** [comment]
Yeah you're right. There's a lot of things not in the docs yet, unfortunately... So much work to keep it all in sync, and clearly worded.

**Comment by koenhandekyn** [comment]
no worries.
examples cover a lot and are easier to maintain and test :) !

i‘ve been playing a couple of days with paper js for a small hobby project and my compliments to you guys
the API feels very very well designed !
good API’s are self explanatory and until now it’s been just the case !
